addons/core/tests/integration/components/form/authenticate-test.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, find } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import { setupIntl } from 'ember-intl/test-support';
+
+module('Integration | Component | form | authenticate', function (hooks) {
```

Comment:
Could we add a some more tests/asserts for this form? My thoughts are with the current test, we could delete the 2 input fields and this tests would still pass. My suggestion would be to assert the inputs and form buttons are present.